test/auction_accounts.js

```diff
@@ -2,14 +2,20 @@ const BatchAuction = artifacts.require("BatchAuction")
 const ERC20 = artifacts.require("ERC20")
 const MintableERC20 = artifacts.require("./ERC20Mintable.sol")
 
+const zeroHash = "0x0"
+const oneHash = "0x1"
```

Comment:
ubernit: when would you use zero vs. one hash?

Reply:
oneHash is used for something generic which is explicitly non-zero (to get past requirements such as `require(hash != 0)`